### PR TITLE
Introduce @ledgerhq/cryptoassets

### PR DESCRIPTION
**Pull request by gre**

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -0,0 +1 @@
+packages/cryptoassets/data
```
Comment (meriadec) [inline .eslintignore:1]: :rage1:

File: packages/cryptoassets/package.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+    "name": "@ledgerhq/cryptoassets",
+    "version": "5.23.0",
+    "description": "Ledger common errors",
```
Comment (meriadec) [inline packages/cryptoassets/package.json:4]: probably not the right description

File: .eslintignore (hunk 1 above)
Comment (gre) [reply .eslintignore:1]: yeah it's generated data, it's valid javascript. but eslint don't like `[,,,,stuff]` which is handy to save bytes.

File: packages/cryptoassets/package.json (hunk 2 above)
Comment (gre) [reply packages/cryptoassets/package.json:4]: true

File: .eslintignore (hunk 1 above)
Comment (meriadec) [reply .eslintignore:1]: i was referring to the "missing EOL char" ^^ not so important though. usually it's better for git diff

**Pull request merged**